### PR TITLE
Fixing build errors

### PR DESCRIPTION
**Pull request by yashwanthreddyg**
Issue was due to duplicate attr name from fabsmenu library and unescaped apostrophes

### CODE REVIEW

File: app/src/main/res/values-is/strings.xml [hunk 1]
```diff
@@ -334,9 +334,9 @@
                         <b>Virkjaðu skráadeilingu á Windows</b>\n
                     </li>
                     <p style='margin-left: 40px'>
-                        Opnaðu stjórnskjáinn (Control Panel), smelltu á 'Choose homegroup and sharing
```
Comment (EmmanuelMess) [inline app/src/main/res/values-is/strings.xml]: You seem to have deleted a few "á" here.

File: app/build.gradle [hunk 2]
```diff
@@ -180,7 +180,7 @@ dependencies {
     }
     annotationProcessor "com.github.bumptech.glide:compiler:$glideVersion"
 
-    implementation 'com.github.jahirfiquitiva:FABsMenu:1.1.3'//Floating Action Buttons Menu (aka expandable FAB)
+    implementation 'me.jahirfiquitiva:FABsMenu:1.1.1'//Floating Action Buttons Menu (aka expandable FAB)
```
Comment (EmmanuelMess) [inline app/build.gradle]: Why did you downgrade from 1.1.3 to 1.1.1?

Comment (yashwanthreddyg) [reply app/build.gradle]: It was a mistake. 1.1.3 had a fabSize attribute conflict. Upgraded it to 1.1.4.

File: app/src/main/res/values-is/strings.xml (hunk 1 above)
Comment (yashwanthreddyg) [reply app/src/main/res/values-is/strings.xml]: Fixed this.

**Pull request closed without merging**